src/metabase/driver/sql/query_processor.clj

```diff
@@ -116,15 +116,22 @@
                                   (- offset) :day)
       (truncate-fn expr))))
 
-(defn adjust-day-of-week
+(s/defn adjust-day-of-week
```

Comment:
just curious, is there a benefit to schema's defn when you provide no type information on return or parameters?

Reply:
nevermind i missed it was annotated in the second arity